logspout/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# uses ONBUILD instructions described here:
+# https://github.com/gliderlabs/logspout/tree/master/custom
+
+FROM gliderlabs/logspout:master
```

Comment:
So the whole point is to include the "github.com/looplab/logspout-logstash" plugin, right?